.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,130 @@
+### Description
+
+<!-- Describe your changes in detail -->
+
+### Motivation & Context
+
+<!-- Why is this change required? What problem does it solve? -->
+
+### Types of changes
+
+<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Refactoring (non-breaking change)
+- [ ] Breaking change (fix or feature that would change existing functionality)
+
+### Checklist
+
+<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+
+- [ ] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
+- [ ] My code follows the code style of the project _(using `npm run lint`)_
+- [ ] My change introduces changes to the documentation
+- [ ] I have updated the documentation accordingly
+- [ ] I have added tests to cover my changes
+- [ ] All new and existing tests passed
+
+
+<!-- /!\ Core Team only: Uncomment the following for a feature DoD -->
+<!--
+### Development
+
+- [ ] Should match specs (eg. either the Web UI Kit or any pattern from the WAI — or both…)
+- [ ] Docs added:
+  - including the "Sass" part using `scss-docs` shortcode
+  - in /about/overview/#custom-components if it is a new Orange custom component
+  - in /getting-started/introduction/#components if it is a new Orange custom component that requires JavaScript (and Popper)
+  - in /customize/overview#csps-and-embedded-svgs if it is a new Orange custom component that includes embedded SVGs in our CSS
+  - in /forms/validation/?#supported-elements if it is a new Orange custom component that is a form control
+  - in /forms/overview/ if it is a new Orange custom component that is a form control
+- [ ] Check (and fix) RTL version
+- [ ] Run linters
```

Comment:
Maybe remind the commands to run by adding :  _(using `npm run ..., npm run ..., etc.`)_